.changeset/pretty-lamps-develop.md

```diff
@@ -0,0 +1,8 @@
+---
+'@vercel/edge-config': patch
```

Comment:
do we actually need to release a patch for edge-config? no point in bumping the version if we didn't make changes

<img width="468" alt="image" src="https://user-images.githubusercontent.com/1765075/235493742-3938f8a3-5a0a-4f64-99a2-cff1f45366c1.png">

Reply:
No

Reply:
haha sorry, wasn't actually asking - it's just my polite way of saying "this should probably not be here"

Reply:
Actually, we need to publish it to update the GitHub links on NPM -- they'll point to a nonexistent repo if we don't.